src/NuGet.Commands/RestoreCommand.cs

```diff
@@ -27,6 +29,9 @@ namespace NuGet.Commands
 {
     public class RestoreCommand
     {
+        // Temporary until we have RESX and loc
```

Comment:
There are a bunch of strings in our trace output we'll also need to consider, especially if they end up in the Output Window...